manager-bundle/src/ContaoManager/ApiCommand/SetConfigCommand.php

```diff
@@ -42,10 +42,10 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $data = @json_decode($input->getArgument('json'), true);
-
-        if (null === $data) {
-            throw new \RuntimeException('Invalid JSON: '.json_last_error_msg());
+        try {
+            $data = json_decode($input->getArgument('json'), true, 512, JSON_THROW_ON_ERROR);
+        } catch (\JsonException $e) {
+            throw new \RuntimeException('Invalid JSON: '.$e->getMessage());
         }
```

Comment:
```suggestion
        $data = json_decode($input->getArgument('json'), true, 512, JSON_THROW_ON_ERROR);
```

I don’t think throwing a custom `RuntimeException` is necessary here.

Reply:
Not sure about that. @aschempp Does the Contao Manager require a particular exception (or a particular message) or just any exception?

Reply:
AFAIK the `config:set` command does not yet get used by the manager.